build/docker/bats/Dockerfile

```diff
@@ -0,0 +1,66 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM alpine:3.8
+
+RUN apk add --no-cache \
+  bash \
+  curl \
+  git \
+  jq \
+  make
+
+ENV APP_BASE_DIR="/cftk"
+# Directory structure used for tools and workspace.
+RUN mkdir -p "$APP_BASE_DIR/home" \
+  "$APP_BASE_DIR/bin" \
+  "$APP_BASE_DIR/workdir"
+
+
+# bats
+WORKDIR /cftk
```

Comment:
`$APP_BASE_DIR` should be used consistently in place of `/cftk`.

Reply:
I initially had this as an inline RUN `cd` but `make check_docker` complained about using WORKDIR, which then complained again about the following.  Should we disable DL3000?

```
make check_docker
Running hadolint on Dockerfiles
./build/docker/bats/Dockerfile:37 DL3000 Use absolute WORKDIR
```

Reply:
Replaced /cftk with $APP_BASE_DIR . in 7e56437...0f74e0d HEAD -> test_structure (forced update)